platform/pulpcore/app/viewsets/repository.py

```diff
@@ -51,6 +51,19 @@ def importers(self, request, name):
         serializer = ImporterSerializer(page, many=True, context={'request': request})
         return paginator.get_paginated_response(serializer.data)
 
+
+    def update(self, request, partial=True):
+        instance = self.get_object()
+        serializer = self.get_serializer(instance, data=request.data, partial=partial)
+        serializer.is_valid(raise_exception=True)
+        async_result = tasks.repository.update.apply_async_with_reservation(
+            tags.RESOURCE_REPOSITORY_TYPE, instance.name,
```

Comment:
I think we should start reserving by repo id instead of repo name. This is necessary if repo names can be changed, which we haven't discussed disallowing yet. Thoughts, @bmbouter?